cumulusci/tasks/metadata_etl/base.py

```diff
@@ -305,3 +306,48 @@ def _transform(self):
                 removed_api_names.add(api_name)
 
         self.api_names = self.api_names - removed_api_names
+
+
+class UpdateMetadataFirstChildTextTask(MetadataSingleEntityTransformTask):
+    """
+    Updates the metadata's first child tag's text.
+    Does not support tags with complex/object-like values.
+    """
+
+    task_options = {
+        "entity": {"description": "Metadata Entity", "required": True},
+        "tag": {
+            "description": "Targeted tag. The text of the first instance of this tag within the metadata entity will be updated.",
+            "required": True,
+        },
+        "value": {
+            "description": "Desired value to set for the targeted tag's text. This value is namespace-injected.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+        self.entity = self.options.get("entity")
+        self.options["value"] = self._inject_namespace(self.options.get("value"))
+
+    def _transform_entity(
+        self, metadata: MetadataElement, api_name: str
+    ) -> MetadataElement:
+        """
+        Finds metadata's first child with tag.  If no child is found, appends
+        a new child with tag.  Then updates child's text as the value option.
+        """
+        tag = self.options.get("tag")
```

Comment:
Within here, it's safe to do `self.options["tag"]` instead of `self.options.get("tag")` because the options are marked as required.

(The framework first calls, _init_options, then BaseTask._validate_options which checks for required fields, then runs the task. So using `get` _is_ necessary in _init_options)

Reply:
Done. Will push momentarily.